docs/webhooks/Getting-Started.md

```diff
@@ -10,21 +10,21 @@ To test out webhooks locally without deploying an application to an environment
 
 Download ngrok [here](https://ngrok.com/download), then follow their getting started guide.
 
-## Configuring webhooks through the portal
+## Configuring webhooks through the dashboard
 
-Once you’re up and running with ngrok you will need to configure your webhooks through our portal, using the left menu to navigate to the webhook pages
+Once you’re up and running with ngrok you will need to configure your webhooks through our dashboard, using the left menu to navigate to the webhook pages
 
-1. Navigate to the webhooks secret page. Click on 'Generate secret' and save its value for later. Note that you may only have two active secrets.
+1. Navigate to the webhooks configuration section and click on the `+` button. Select your desired secret lifetime and then click on 'Create secret' and save its value for later. Note that you may only have two active secrets.
```

Comment:
```suggestion
1. Navigate to the webhooks configuration section and click on the `+` button in the secrets box/section/tile. Select your desired secret lifetime and then click on 'Create secret' and save its value for later. Note that you may only have two active secrets.
```
it's not fully clear where the '+' button is imo

Reply:
That card in the UI is called `Configuration`, secrets is a sub-heading. Would `click on the '+' button next to Secrets heading` be clearer?

Reply:
ah I see, yeah that would be clearer